platform/ui/src/elements/form/DropdownMenu.css

```diff
@@ -69,6 +69,7 @@
   color: var(--text-color-active);
   padding: 10px 15px;
   border-bottom: 1px solid #ccc;
+  width: 100%;
```

Comment:
Are we using this anywhere else? Did we test to make sure this didn't negatively impact other usage?

Reply:
This was fixed with the latest changes. thx.